### DISCUSSION TITLE
Information about ERT field data with topography

### DISCUSSION THREAD

**Discussion by bralec** [gimli-org/pyGIMLi#496] [General]
Hello again, I hope everything is going very well.

I would like to know if by any chance information about the data measured in example https://www.pygimli.org/_examples_auto/3_dc_and_ip/plot_02_ert_field_data.html#sphx-glr-examples-auto-3-dc-and-ip-plot-02-ert-field-data-py, apart from the fact that it was in a slag dump.

Best regards.

**Comment by halbmy** [comment]
Yes, the profile is one of the profiles presented by Günther et al. (2006). See Figure 10 for the location, it is the rightmost of the cross profiles. There is some text in Figure 4, but of course there is more information on the slag heap available. The data was measured by my colleagues Ursula Noell and Markus Furche from BGR Hannover in 2004. I can ask them for a report or additional information and I still have the data of all profiles.

**Comment by bralec** [comment]
Excellent! Thank you very much for the information and the document corresponding to the data, it was more than enough for what I need. Although, it would be very useful for me to have the data of the 13 profiles, as you mention, thank you very much in advance.

**Reply by halbmy** [reply]
I removed the pdf file as it could violate copyrights. Please try https://dx.doi.org/10.1111/j.1365-246X.2006.03011.x or request a copy on https://www.researchgate.net/publication/227663284_Three-dimensional_modelling_and_inversion_of_DC_resistivity_data_incorporating_topography_-_II_Inversion

**Reply by bralec** [reply]
Ohh ok, I understand, an apology, I'll be more careful in the future with that.

**Comment by halbmy** [comment]
The full data set is available in the example data under https://github.com/gimli-org/example-data/blob/master/ert/slagdump3d.ohm
